src/aggregations/bucket-aggregations/composite-agg-values-sources/values-source-base.js

```diff
@@ -105,6 +105,17 @@ class ValuesSourceBase {
         return this;
     }
 
+    /**
+     * Specifies to include documents without a value for a given source in the
```

Comment:
Change to `Specifies whether to include documents without a value for a given source in the response. Defaults to `false` (not included).`. Could you also wrap the comment at 80 characters?

Reply:
Addressed in https://github.com/sudo-suhas/elastic-builder/pull/108/commits/96fc5174f33a364a69dbf914da499b1f57cff1de